src/test/java/uk/gov/hmcts/darts/task/runner/impl/AudioLinkingAutomatedTaskTest.java

```diff
@@ -159,14 +156,12 @@ void positiveProcessEvent() {
         }
 
         @Test
-        void positiveProcessEventWithBuffer() {
-            doNothing().when(eventProcessor)
-                .processMedia(any(), any(), any());
+        void processEvent_withBuffer() {
```

Comment:
Suggest we align to the new [test naming standard](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1694706043#BestPractices(Backend)-TestNamingShallClearlyCommunicatetheTestIntent).